createCoursesFile.js

```diff
@@ -172,20 +114,17 @@ function filterCoursesDuringPeriod (arrayOfCourseRoundArrays, period) {
   return arrayOfCourseRoundArrays.map(arrayOfCourseRounds => arrayOfCourseRounds.filter(({periods}) => periods && periods.find(({number}) => number === period)))
 }
 
-
-module.exports = function ({term, year, period}) {
-  const termin = `${year}:${term}`
-  const fileName = `csv/courses-${termin}-${period}.csv`
-  console.log('Using file name:', fileName)
-  return deleteFile(fileName)
+module.exports = {
+  createCoursesFile ({term, year, period}) {
+    const termin = `${year}:${term}`
+    const fileName = `csv/courses-${termin}-${period}.csv`
+    console.log('Using file name:', fileName)
+    return deleteFile(fileName)
     .then(() => getCourseRoundsPerCourseCode(termin))
-    .then(courses => {
-      console.log('courses', JSON.stringify(courses, null, 4))
-      return courses
-    })
-    .then(courses=>filterSelectedCourses(courses))
+    .then(courses => filterSelectedCourses(courses))
```

Comment:
No, we wont need it when we remove the logic from create courses. But that isn't in the scope of this feature or sprint, our goal now is just to start using sections so that we in the near future can remove all the logic about filtering courses